sequencer/finalizer.go

```diff
@@ -667,11 +676,7 @@ func (f *finalizer) processForcedBatch(ctx context.Context, lastBatchNumberInSta
 	f.nextGERMux.Lock()
 	f.lastGERHash = forcedBatch.GlobalExitRoot
 	f.nextGERMux.Unlock()
-
-	err = f.handleForcedBatchProcessResp(request, response, stateRoot)
-	if err != nil {
-		return lastBatchNumberInState, stateRoot
-	}
+	_ = f.handleForcedTxsProcessResp(request, response, stateRoot)
```

Comment:
why do we ignore the error?

Reply:
because nothing happens even if we have error the I can just remove returning it

Reply:
because nothing happens even if we have error I can just remove returning it